modules/rhdh-plugins-reference/ref-rhdh-plugins.adoc

```diff
@@ -1,5 +1,5 @@
 [id="rhdh-supported-plugins"]
-== Dynamic plugins included in {product}
+= Preinstalled dynamic plugin descriptions and details
```

Comment:
Might "Preinstalled" give the impression that the plugins are installed and enabled? I know you discussed this with SME, so I'm happy to go with whatever terminology you decided was most appropriate.

Reply:
Good Q! I made this change because the IBM style guide says to use "preinstalled" instead of "pre-loaded" and I wanted us to be compliant, but I do see what you mean. I did talk to both SMEs about these terms because the difference between "included" and "preinstalled" in not clear. Rather than rely on the nuances between "preinstalled" and "included" to communicate preinstalled and enabled/disabled by default, we are starting to more clearly and directly say what these words mean, which is also helpful for translation.

Reply:
In multiple places, we say that plugins are enabled or disabled by default, but we should prob include this info even more directly in `ref-rhdh-plugins` and 'con-preinstalled-dynamic-plugins`